pkg/nodeconfig/nodeconfig.go

```diff
@@ -255,25 +249,40 @@ func (nc *nodeConfig) configureNetwork() error {
 	return nil
 }
 
-// addVersionAnnotation adds the version annotation to nc.node
-func (nc *nodeConfig) addVersionAnnotation() {
-	nc.node.Annotations[VersionAnnotation] = version.Get()
-}
-
-// addPubKeyHashAnnotation adds the public key annotation to nc.node
-func (nc *nodeConfig) addPubKeyHashAnnotation() {
-	nc.node.Annotations[PubKeyHashAnnotation] = nc.publicKeyHash
+// addIdentifyingAnnotations adds the public key hash annotation as well as any additional annotations
+// that could help identify what type of node is being configured
+func (nc *nodeConfig) addIdentifyingAnnotations() error {
```

Comment:
This obfuscates what we are doing and a bit confusing. `could help` implies it may or may not help. I would rather see `nc.addAnnotations()` being called at the call site with a map of annotations and this function removed.

Reply:
Agreed. I have addressed this.